schemas/com.snowplowanalytics.snowplow/media_player_ad_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a media event related to ads during the media playback",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_ad_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "type": {
```

Comment:
I would go so far as to separate this out into different schemas entirely (e.g., ad_start, ad_end etc) so that we can capture the properties separately.

Reply:
Right, I separated out the individual event schemas into another PR – #1283
There are quite a few of them, let me know if there is something I can do to make it easier to review!